students/BRACH_Jakub/lesson_01_basics/L01P19_ClockFace.py

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env python3
+h = int(input())
+m = int(input())
+s = int(input())
+
+angle = h * 30 + m * (30 / 60) + s * (30 / 3600)
```

Comment:
In general it's better to write `60 * 60 ` than `3600`. This way is more readable and shows how some values are built.